Dockerfile

```diff
@@ -28,16 +29,21 @@ SHELL ["/bin/bash", "-c"]
 # Install commands used in our scripts and standard present on a clean ubuntu
 # installation and setup a user with sudo priviledges
 RUN apt-get update -qq && \
-    apt-get install -qq --assume-yes --no-install-recommends apt-transport-https apt-utils ca-certificates curl dbus dialog git lsb-release sudo tzdata wget && \
+    apt-get install -qq --assume-yes --no-install-recommends apt-transport-https apt-utils ca-certificates curl dbus dialog git lsb-release openssh-client sudo tzdata wget > /dev/null && \
     # Add amigo user
-    adduser --disabled-password --gecos "" $USER && \
-    adduser $USER sudo && \
-    echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+    adduser -u 1000 --disabled-password --gecos "" $USER && \
+    usermod -aG sudo $USER && \
+    usermod -aG adm $USER && \
+    echo "%sudo ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/"$USER"
 
 # Setup the current user and its home directory
 USER "$USER"
 WORKDIR /home/"$USER"
 
+ADD installer/bootstrap.bash ./bootstrap.bash
```

Comment:
I am not sure I prefer this over the old solution

Reply:
There's no difference between the two solutions. This would work in private gitlab instances too while the other would require a lot of configuration.